guides/common/modules/proc_configuring-simplified-alternate-content-sources.adoc

```diff
@@ -0,0 +1,39 @@
+[id="Configuring_Simplified_Alternate_Content_Sources_{context}"]
+= Configuring Simplified Alternate Content Sources
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources*.
+. Click *Add Source* and set the *Source type* as *Simplified*.
+. Select the *Content type*.
+. In the *Name* field, enter a name for the alternate content source.
+. Optional: In the the *Description* field, provide a description for the ACS.
+. Select {SmartProxies} to which the alternate content source is to be synced.
+. Optional: Select `Use HTTP proxies` if you want the ACS to use the {SmartProxy}'s HTTP proxy.
+. Select the products whose repositories you want to sync.
```

Comment:
"sync" is a bit of an overloaded term here, I don't wanna get folks confused with repository syncing.  Maybe something higher level, like, "Select the products that should use alternate content sources".